src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -0,0 +1,34 @@
+# # -*- coding: utf-8 -*-
+from typing import Optional
+
+from .. import mgr
+from ..services import feedback
+from . import ApiController, ControllerDoc, RESTController
+from ..model.feedback import Feedback
+from ..security import Scope
+
+@ApiController('/feedback', Scope.CONFIG_OPT)
+@ControllerDoc("Feedback API", "Report")
+class FeedbackController(RESTController):
+    issueAPIkey = None
+
+    def __init__(self):  # pragma: no cover
+        super(FeedbackController, self).__init__()
+        self.cephTrackerClient = feedback.CephTrackerClient()
+
+    def create(self, project_id, tracker_id, subject, description, severity: Optional[str] = '3 - Minor'):
```

Comment:
```suggestion
    @RESTController.MethodMap(version='0.1')
    def create(self, project_id, tracker_id, subject, description, severity: Optional[str] = '3 - Minor'):
```

As this is a new, evolving feature potentially subject to changes, let's mark it as experimental (version='0.1').

Reply:
Yes